### PR TITLE
Heroku Connector article

### PR DESCRIPTION
**Pull request by kayla-broussard**
As requested by @dallasread, here's a first draft of a support article for the Heroku connector. It covers all steps required, and also lets customers know where they can view the metadata or remove the connector from their domain.

### CODE REVIEW

**Review by deandre — APPROVED** [review]
LGTM!

**Comment by dallasread** [comment]
@deandre As far as I know, they will both exist for the time being with gentle nudges toward the Connector. Generally speaking, I expect there will be some sort of migration from 1-click to addons.

File: content/articles/heroku-connector.markdown [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+1. Head into the "connections" tab for the domain you want to connect to Heroku. From there, click the "add" button next to the Heroku connector.  
+   ![Add the connector](/files/heroku-connector-add-connector.png)
+
+
+2. Authorize your DNSimple account to connect to Heroku.  
+   ![Authorize your DNSimple account](/files/heroku-connector-authenticate-dnsimple.png)
+   
+
+3. Authorize the connection to Heroku.
```
Comment (dallasread) [inline content/articles/heroku-connector.markdown:29]:
```suggestion
3. Authorize your Heroku account to give the connector permission to access your Heroku apps.
```

File: content/articles/heroku-connector.markdown [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+1. Head into the "connections" tab for the domain you want to connect to Heroku. From there, click the "add" button next to the Heroku connector.  
+   ![Add the connector](/files/heroku-connector-add-connector.png)
+
+
+2. Authorize your DNSimple account to connect to Heroku.  
+   ![Authorize your DNSimple account](/files/heroku-connector-authenticate-dnsimple.png)
+   
+
+3. Authorize the connection to Heroku.  
+![Authorize the connection to Heroku](/files/heroku-connector-connect-heroku.png)
```
Comment (dallasread) [inline content/articles/heroku-connector.markdown:30]:
```suggestion
![Authorize your Heroku account](/files/heroku-connector-connect-heroku.png)
```

File: content/articles/heroku-connector.markdown [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+   
+
+4. Specify which Heroku app you want to connect to the domain and the mounting point.
```
Comment (dallasread) [inline content/articles/heroku-connector.markdown:33]:
```suggestion
4. Choose your Heroku app and the set its mounting point.
```

File: content/articles/heroku-connector.markdown [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+* TOC
+{:toc}
+
+---
+
+[Heroku](https://www.heroku.com) is a platform as a service (PaaS) that enables developers to build, run, and operate applications entirely in the cloud. With DNSimple's Heroku Connector, you can easily connect your domains to Heroku and set up the required DNS records.
+
+## Using the connector
+
+1. Head into the "connections" tab for the domain you want to connect to Heroku. From there, click the "add" button next to the Heroku connector.
```
Comment (dallasread) [inline content/articles/heroku-connector.markdown:21]:
```suggestion
1. Head into the "Connections" tab for the domain you want to connect to Heroku. From there, click the "Add" button next to the Heroku connector.  
```

File: content/articles/heroku-connector.markdown [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+{:toc}
+
+---
+
+[Heroku](https://www.heroku.com) is a platform as a service (PaaS) that enables developers to build, run, and operate applications entirely in the cloud. With DNSimple's Heroku Connector, you can easily connect your domains to Heroku and set up the required DNS records.
+
+## Using the connector
+
+1. Head into the "connections" tab for the domain you want to connect to Heroku. From there, click the "add" button next to the Heroku connector.  
+   ![Add the connector](/files/heroku-connector-add-connector.png)
```
Comment (dallasread) [inline content/articles/heroku-connector.markdown:22]:
```suggestion
   ![Create a connection to Heroku](/files/heroku-connector-add-connector.png)
```

File: content/articles/heroku-connector.markdown [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+   ![Authorize your DNSimple account](/files/heroku-connector-authenticate-dnsimple.png)
+   
+
+3. Authorize the connection to Heroku.  
+![Authorize the connection to Heroku](/files/heroku-connector-connect-heroku.png)
+   
+
+4. Specify which Heroku app you want to connect to the domain and the mounting point.
+   <info>
+   A mount point is the address where your Heroku app is served. If you want to use the domain apex, e.g. `example.com`, leave this blank.
```
Comment (dallasread) [inline content/articles/heroku-connector.markdown:35]:
```suggestion
   A mounting point is the address where your Heroku app is served. If you want to use the domain apex, e.g. `example.com`, leave this blank.
```

File: content/articles/heroku-connector.markdown [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+   </info>  
+   ![Connect your domain to Heroku](/files/heroku-connector-connect-domain.png)
+   
+
+5. You will be asked to review the connection configuration, and ensure the information is correct, by hitting "connect".
```
Comment (dallasread) [inline content/articles/heroku-connector.markdown:40]:
```suggestion
5. You will be asked to review the connection configuration. To confirm the information is correct, click "connect".   
```

**Review by itsalyse — APPROVED** [review]

**Pull request merged**